lib/DDG/Spice/JrDevJobs.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::JrDevJobs;
+# ABSTRACT: Search for Junior Developer Jobs from JrDevJobs.com
+
+use DDG::Spice;
+
+triggers start => qw(jrdevjobs);
+
+spice to => 'https://www.jrdevjobs.com/jobs.json?query=$1';
+spice is_cached => 1;
+spice wrap_jsonp_callback => 1;
+
+handle remainder => sub {
+   s/(jobs)//;
+   s/(job)//;
```

Comment:
We can simplify this a bit: `s//jobs?//` will match "job" and "jobs" because the `s?` makes the "s" optional.